src/components/StudyPane/InfoPane/Motif/Root.tsx

```diff
@@ -34,25 +35,22 @@ const Root = ({
         });
     })
     const handleMouseDown = (event: React.MouseEvent<HTMLDivElement>) => {
-        event.preventDefault();
-        event.stopPropagation();
-        document.addEventListener('mouseup', handleMouseUp);
+        // event.preventDefault();
+        // event.stopPropagation();
         const target = event.target as HTMLElement;
         const clickedTarget = target.getAttribute('data-clickType');
         clickedTarget == "clickable" ? setClickToDeSelect(false) : setClickToDeSelect(true);
-      };
 
-      const handleMouseUp = useCallback((event: MouseEvent) => {
-        //document.body.style.userSelect = 'text';
         if (clickToDeSelect) {
-          ctxSetSelectedRoots([]);
-          const filteredHebWords = ctxSelectedHebWords.filter(hebWord => 
-            !ctxSelectedRoots.includes(hebWord.strongNumber)
-          );
-          ctxSetSelectedHebWords(filteredHebWords)
-          console.log(filteredHebWords)
+            ctxSetSelectedRoots([]);
+            const filteredHebWords = ctxSelectedHebWords.filter(hebWord => 
+              !ctxSelectedRoots.includes(hebWord.strongNumber)
+            );
+            ctxSetSelectedHebWords(filteredHebWords)
+            console.log(filteredHebWords)
         }
-      }, [clickToDeSelect]);
```

Comment:
Why took this out?

Reply:
took out the entire handleMouseUp function under Root.tsx because it messes up drag-to-select's handleMouseUp function and caused the bug